open_event/helpers/oauth.py

```diff
@@ -0,0 +1,69 @@
+class OAuth(object):
+    """Google Credentials"""
+
+    CLIENT_ID='449612261522-1eg34prt23l0454et59qgqno3rjd8muq.apps.googleusercontent.com'
+    CLIENT_SECRET='aq8XaUlxCfhwwMyZyNw8kS-D'
```

Comment:
The secret should be made into a travis secret: https://docs.travis-ci.com/user/encryption-keys/